### DISCUSSION TITLE
Cannot connect to Apple Developer Account

### DISCUSSION THREAD

**Discussion by Fabian010697** [codemagic-ci-cd/codemagic-docs#1099] [Q&A] [answered]
Hello everybody!

I want to connect my Apple Developer Account, but in the user preferences is no option to do this.

![image](https://user-images.githubusercontent.com/46001656/198286292-df2f9b3d-7bff-45f8-ba59-bb06a0080505.jpeg)

**Comment by himesh-cm** [comment, marked as answer]
Hi @Fabian010697, This has been moved to the team's section. 
Let me know if you face any issues.
